packages/dom/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.0.9 (Unreleased)
+
+- Update `isHorizontalEdge` to account for empty text nodes.
```

Comment:
I think this is missing headings. `### Enhancement`...

Reply:
Sorry, I followed previous formatting. I'll adjust